images/router/haproxy/conf/haproxy-config.template

```diff
@@ -208,9 +208,9 @@ backend be_edge_http_{{$cfgIdx}}
   http-request set-header X-Forwarded-Proto http if !{ ssl_fc }
   http-request set-header X-Forwarded-Proto https if { ssl_fc }
   {{ if (eq $cfg.TLSTermination "") }}
-    cookie OPENSHIFT_{{$cfgIdx}}_SERVERID insert indirect nocache httponly
+    cookie {{env "COOKIE_PREFIX_MAIN" "OPENSHIFT_"}}{{$cfg.Hash}}{{env "COOKIE_SUFFIX" "_SERVERID"}} insert indirect nocache httponly
```

Comment:
Just `cookie {{$cfg.Hash}}` here. Might be clearer as well to rename `Hash` to `RoutingKeyName` - that it is a hash/obfuscated depends on the flag inside of the router code.